triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceMain.scala

```diff
@@ -62,14 +61,15 @@ case class LedgerConfig(
 final case class RunningTrigger(
     triggerInstance: UUID,
     triggerName: Identifier,
-    jwt: Jwt,
+    token: String, // User credentials.
+    // TODO(SF, 2020-): Add access token field here in the presence of authentication.
     runner: ActorRef[TriggerRunner.Message]
 )
 
 class Server(dar: Option[Dar[(PackageId, Package)]], triggerDao: Option[TriggerDao]) {
 
   private var triggers: Map[UUID, RunningTrigger] = Map.empty;
-  private var triggersByToken: Map[Jwt, Set[UUID]] = Map.empty;
+  private var triggersByToken: Map[String, Set[UUID]] = Map.empty;
```

Comment:
I think it would make sense for this map to have username keys and then have a separate map for passwords if we are storing them like that. Then we obtain the password only when necessary, i.e. before calls to the auth service.

Reply:
I don't see any great benefit to this over doing what we currently do. I do agree with the introduction of a case class `UserCredentials (token: String)` you suggest above.

Reply:
If you use an encrypted token then it probably doesn't matter too much. Otherwise it is for the better hygiene of not passing around unencrypted passwords.

Reply:
we are agreed that we will encrypt the credential tokens. do you feel that the encryption should go in on this PR or would you be ok if it went in on the PR after this one?